wandb/sdk/internal/sender.py

```diff
@@ -1567,6 +1567,8 @@ def _flush_job(self) -> None:
                 proto_artifact.run_id = self._run.run_id
                 proto_artifact.project = self._run.project
                 proto_artifact.entity = self._run.entity
+                for alias in ["latest"]:
+                    proto_artifact.aliases.append(alias)
```

Comment:
Why not just `proto_artifact.aliases.append("latest")`? I'd probably not add the for loop until there was more than one.

Reply:
Fixed